src/lib/envelope/envelope.ml

```diff
@@ -1,8 +1,12 @@
 open Core
 open Network_peer
 
+module Sender = struct
+  type t = Local | Remote of Peer.t [@@deriving sexp, bin_io]
```

Comment:
since this is `bin_io`, should it be versioned?

Reply:
good catch